packages/client/src/__tests__/cloudFunctions.test.ts

```diff
@@ -517,7 +517,8 @@ describe("Cloud functions", () => {
     const minimalSaul: CustomerBase = {
       name: saul.name,
       surname: saul.surname,
-      email: saul.email,
+      // non-normalized email
+      email: " SAUl@Gmail.cOm ",
```

Comment:
Nitpick: it's nice to have a single source of truth. We could use `saul.email.replace(/.*@|o|g/g, (match) => match.toUpperCase());` here and get the same result. If the email changes the part before the `@` sign will still match, and will still be uppercase. Disclaimer: [chatgpt wrote](https://chat.openai.com/share/3825208f-3a18-40a6-8ecd-8b39bc98f625) a first incarnation of that code.